rules/prefer-node-append.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const getMethodName = memberExpression => memberExpression.property.name;
+
+const fix = (fixer, sourceCode, callNode, memberExpression) => {
+	const nodeCode = sourceCode.getText(memberExpression.object);
+	const argumentsCode = sourceCode.getText(callNode.arguments[0]);
+	const fixedCodeStatement = `${nodeCode}.append(${argumentsCode})`;
+	return fixer.replaceText(callNode, fixedCodeStatement);
+};
```

Comment:
This could be simplified a lot. Don't get the source code and construct it manually. Just replace the property at the node level.

Something like:

```js
fixer.replaceText(memberExpression.property, 'append');
```